### PR TITLE
Duplicate longitude wrap

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

File: Packages/vcs/Lib/vcs2vtk.py [hunk 1]
```diff
@@ -609,6 +609,8 @@ def project(pts, projection, wc, geo=None):
         pname = projDict.get(projection._type, projection.type)
         projName = pname
         pd.SetName(projName)
+        pd.SetOptionalParameter("over", "true")
+        pd.SetCentralMeridian(float(xm + xM) / 2.0)
```
Comment (doutriaux1) [inline Packages/vcs/Lib/vcs2vtk.py]: @danlipsa what about case where user specified the central meridian?

Comment (danlipsa) [inline Packages/vcs/Lib/vcs2vtk.py]: I just seen your post about this. I'll look at it and use that value instead.

Comment (aashish24) [inline Packages/vcs/Lib/vcs2vtk.py]: > @danlipsa what about case where user specified the central meridian?

@doutriaux1 that what was the question @danlipsa asked in the issue. How does someone specify that in vcs?

Comment (doutriaux1) [inline Packages/vcs/Lib/vcs2vtk.py]: @aashish24 I replied to @danlipsa he's on it. Thx.

**Comment by aashish24** [comment]
thanks @doutriaux1 

@danlipsa when you make a pull request which has new baselines, could you please reference to the data pull request as well? 

https://github.com/UV-CDAT/uvcdat-testdata/pull/96

**Comment by aashish24** [comment]
@doutriaux1 once this and the clipped data (and two of yours merged in) we should finalize the release.. what you think?

**Comment by doutriaux1** [comment]
@sankhesh was VERY close from getting XCode 7 in, I would like to have this in as well!

**Comment by aashish24** [comment]
@danlipsa some tests arw failing. Are you lookinf intonit?

**Comment by danlipsa** [comment]
@aashish24 @sankhesh @doutriaux1 This is strange. For some machines all test pass, for some there are 85 tests failing. I don't see what is different about the machines that fail. Any ideas?

**Comment by doutriaux1** [comment]
weird I was going to say "MESA" but no. Very odd. But most machines fail though.

**Comment by sankhesh** [comment]
I'm compiling on my Mac to see if I can reproduce those issues. Hopefully I can.

**Comment by danlipsa** [comment]
At least on https://open.cdash.org/viewTest.php?onlyfailed&buildid=4151636
seems to be a problem with the baselines
https://open.cdash.org/testDetails.php?test=402903788&build=4151636
has the wrong baseline.

**Comment by danlipsa** [comment]
@aashish24 @doutriaux1 Thanks @sankhesh for building uvdat on your mac laptop and duplicating the problem. It seems that proj4 behaves differently on that build. Removing +over option fixes the problem for the -3 projection. I am building uvcdat on my mac laptop to debug the proj4 behavior.

**Comment by danlipsa** [comment]
@aashish24 @sankhesh @doutriaux1 I have a hard time building uvcdat on my mac. One problem is that cdtime and cdms2 tried to link against libpng15 which did not exist on my system. I fixed this by changing the link command to link to libpng. Now I get:

ImportError: dlopen(/Users/danlipsa/uvcdat/build/install/Externals/lib/python2.7/site-packages/vtk/vtkCommonCorePython.so, 2): Library not loaded: libvtkCommonCorePython27D-6.3.1.dylib
  Referenced from: /Users/danlipsa/uvcdat/build/install/Externals/lib/python2.7/site-packages/vtk/vtkCommonCorePython.so
  Reason: image not found

However that library is in:
ls -l install/Externals/lib/libvtkCommonCorePython27D-6.3.dylib 
lrwxr-xr-x  1 danlipsa  staff  37 Dec 17 19:09 install/Externals/lib/libvtkCommonCorePython27D-6.3.dylib -> libvtkCommonCorePython27D-6.3.1.dylib
Any suggestions?

**Comment by danlipsa** [comment]
OK, I can reproduce the proj4 problem on my laptop. The build problem I was seeing (libvtkCommonCorePython27D) is still there but that appears only when you run the test through runtest. If you run it directly through python it works fine.

**Comment by aashish24** [comment]
> @aashish24 @sankhesh @doutriaux1 I have a hard time building uvcdat on my mac. One problem is that cdtime and cdms2 tried to link against libpng15 which did not exist on my system. I fixed this by changing the link command to link to libpng. Now I get:

@danlipsa you have to install xquartz (https://github.com/UV-CDAT/uvcdat/wiki/Install-on-Mac-OS-X) and then make sure you build from scratch (and remove any workardounds that you may have).

**Comment by danlipsa** [comment]
Thanks @aashish24 I used xwindows installed with mac port. I will try with xquarts.

**Comment by danlipsa** [comment]
@aashish24 @chaosphere2112 meshfill dataset varies from -40 to 361. It is not clear to me why the portion from -40 to 0 is missing from the plot. This may be a bug in proj4. It would be interesting to test this with the new proj4. Aashish, how is upgrading to the new proj4 going?

**Comment by aashish24** [comment]
@danlipsa the mangling will be a problem since I won't have time tomorrow to look into it. the code to transition to newer proj4 is there. We would also need to build proj4 as well.

**Comment by danlipsa** [comment]
@doutriaux1 Are the buildbots ok? They been running for more than 24 hours.

**Comment by doutriaux1** [comment]
I think sometihng got stuck, I'm updating the bots system to timeout.

**Pull request merged**